utilities/scripts/kcp/manifests/workloadcluster.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: workload.kcp.dev/v1alpha1
+kind: WorkloadCluster
+metadata:
+ name: local
+spec:
+ kubeconfig: |
```

Comment:
I was checking the WorkloadCluster API and looks like it no longer has the `kubeconfig` field. When I sync a workload using the `kubectl kcp workload sync` cmd the WorkloadCluster resource is created automatically. So maybe we don't have to manually create this workload resource.

https://github.com/kcp-dev/kcp/blob/f58e582a75b53d6bc5c9ceb4551f18e40f0f240a/pkg/apis/workload/v1alpha1/workloadcluster_types.go#L56